src/main/java/com/clickhouse/kafka/connect/sink/ProxySinkTask.java

```diff
@@ -91,12 +95,43 @@ public void put(final Collection<SinkRecord> records) throws IOException, Execut
 
         statistics.recordProcessingTime(processingTime);
         // TODO - Multi process???
+        List<List<Record>> failedMessages = new ArrayList<>(dataRecords.size());
```

Comment:
Question: If we trigger a stop/restart to the connector, does it wait until the put method is finished?